test/run/test_stages.py

```diff
@@ -123,18 +124,20 @@ def assertMetadata(self, metadata: Dict, result: Dict):
 
     @classmethod
     def setUpClass(cls):
-        cls.cache = tempfile.TemporaryDirectory(dir="/var/tmp")
+        cls.store = os.getenv("OSBUILD_TEST_STORE")
+        if not cls.store:
+            cls.store = tempfile.mkdtmp(prefix="osbuild-test-", dir="/var/tmp")
```

Comment:
`mkdtemp`

Reply:
Oops indeed. I've done my testing with `OSBUILD_TEST_STORE` enabled all of the time and thus never exercised this branch. Happy to move this to a separate pull request ­— it's not technically related.